src/main/scala/scynamo/ScynamoDecoder.scala

```diff
@@ -141,6 +142,13 @@ trait DefaultScynamoDecoderInstances extends ScynamoDecoderFunctions with Scynam
   implicit val durationDecoder: ScynamoDecoder[Duration] =
     longDecoder.map(Duration(_, TimeUnit.NANOSECONDS))
 
+  /** Using a custom formatter because
+    * "Years outside the range 0000 to 9999 must be prefixed by the plus or minus symbol."
+    * but the plus symbol is not added by the default `.toString`.
+    */
+  implicit val yearMonthDecoder: ScynamoDecoder[YearMonth] =
+    stringDecoder.map(YearMonth.parse(_, DateTimeFormatter.ofPattern("uuuu-MM")))
```

Comment:
This should use `ScynamoDecoder.convert` to catch exceptions from `parse`.
Also let's extract the formatter to a value so that we don't create a new one every time.

Reply:
Extracted the formatter and using `ScynamoDecoder.convert` in https://github.com/moia-oss/scynamo/pull/255/commits/4c49d853f644efd20b11fc415b527ba7ff0775be.